apps/studio/src/features/dashboard/components/ResourceTable/ResourceTableMenu.tsx

```diff
@@ -1,5 +1,6 @@
 import { MenuButton, MenuList, Portal } from "@chakra-ui/react"
 import { IconButton, Menu } from "@opengovsg/design-system-react"
+import { useSetAtom } from "jotai"
```

Comment:
I saw your description as to what this is, but can share why do we need this to be an atom with the 3 modes?

Reply:
the 3 modes are provided by `jotai`. atoms are the base building blocks from `jotai` so if we use `jotai`, we have to use the atoms